src/main/java/gregtech/loaders/recipe/MachineRecipeLoader.java

```diff
@@ -830,6 +830,52 @@ private static void registerAssemblerRecipes() {
                         ConfigHolder.recipes.casingsPerCraft))
                 .duration(200).buildAndRegister();
 
+        ASSEMBLER_RECIPES.recipeBuilder().EUt(48).duration(280)
+                .input(plateDouble, Inconel)
+                .input(plate, Steel, 5)
+                .input(frameGt, Steel)
+                .outputs(MetaBlocks.FISSION_CASING.getItemVariant(
+                        BlockFissionCasing.FissionCasingType.REACTOR_VESSEL, ConfigHolder.recipes.casingsPerCraft))
+                .buildAndRegister();
+
+        ASSEMBLER_RECIPES.recipeBuilder().EUt(48).duration(280)
+                .input(pipeLargeFluid, Inconel)
+                .input(frameGt, Steel)
+                .outputs(MetaBlocks.FISSION_CASING.getItemVariant(
+                        BlockFissionCasing.FissionCasingType.COOLANT_CHANNEL))
+                .buildAndRegister();
+
+        ASSEMBLER_RECIPES.recipeBuilder().EUt(48).duration(280)
+                .input(stick, Zircaloy, 6)
+                .input(ring, Zircaloy, 1)
+                .circuitMeta(1)
+                .outputs(MetaBlocks.FISSION_CASING.getItemVariant(
+                        BlockFissionCasing.FissionCasingType.FUEL_CHANNEL))
+                .buildAndRegister();
+
+        ASSEMBLER_RECIPES.recipeBuilder().EUt(48).duration(280)
+                .input(stick, Zircaloy, 3)
+                .input(ring, Zircaloy, 1)
```

Comment:
Technically, all that these control rod channels would be is something that just holds the control rod in place, which requires, from what I've seen in pictures, basically just some rods to hold it in the middle (and some rings to prevent it from getting misaligned). Perhaps the control rod casing could be something other than a full block? But then I would probably die before I finished this PR.